### PR TITLE
FIX:[#58] - (2)

### PR DESCRIPTION
**Pull request by Madhavdua**
Closes: 

- **Title** : UI improvements
- **Name:** Madhav
- SSOC contributor

Give a clear description of what have you added or modifications made
**I have made the below changes with the required bg wave animation and file formatting.**
I have improved UI, Restructured some elements i.e instead of having complete code in one file, seperate components like searchbar.jsx etc.
Fixed negative episode search

### Type of change ☑️

- [ x] Bug fix : **Episode will be searched only for positive value**
- [ x] New feature : **updated UI**
- [ x] Code style update: Restructured some elements i.e instead of having complete code in one file, seperate components like searchbar.jsx etc.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [ ] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/README.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Chanakya-Niti/blob/main/CODE_OF_CONDUCT.md) of this project.
- [ x] This PR does not contain plagiarized content.
- [x ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x ] My changes generate no new warnings.

### Screenshots 📷
Screenshot 2024-06-15 131849
Screenshot 2024-06-15 131913

### Note to reviewers 📄
Instead of passing of props, using context Api should be used.
For which, should i raise an issue for it?

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Madhavdua i think you have not updating your forked repo. That's why there are alots of conflicts in this pr, either resolve them or re-raise your pr after updating your repo.

This makes hard to review the pr.

**Comment by Madhavdua** [comment]
I have carefully resolved all conflicts. Kindly preview them.

File: src/components/Alert/Alert.jsx
Comment (Avdhesh-Varshney) [inline src/components/Alert/Alert.jsx:1]: un-necessary change, remove this new created line

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Madhavdua still there are some files where double quotes is un-necessarly updated by single quotes.

File: src/components/Alert/Alert.jsx
Comment (Avdhesh-Varshney) [reply src/components/Alert/Alert.jsx:1]: work on the ui as it's not good ux.

image

Comment (Madhavdua) [reply src/components/Alert/Alert.jsx:1]: > work on the ui as it's not good ux.
> 
> image

Sir, i don't know why you are getting this UI, i have attached SS of UI i made.Kindly help me understand the issue.
Screenshot 2024-06-15 131849
Screenshot 2024-06-17 003351

**Comment by Madhavdua** [comment]
Kindly update me sir.

**Comment by Avdhesh-Varshney** [comment]
@Madhavdua now it's looks nice. [link](https://chanakya-git-fork-madhavdua-ui-ba42ee-avdheshvarshneys-projects.vercel.app/). But this will remove the contribution of others from the website like quotes generation and many more things.

**Comment by Madhavdua** [comment]
I understand.
The quote section is still present in code and i will display it properly in future updates.
The other major difference is the input (Search box), which i'll be working on to integrate and will soon update the branch.
Kindly tell me any futher requirements (if any).

**Comment by Avdhesh-Varshney** [comment]
> Thank you for submitting your pull request! We'll review it as soon as possible. For further communication, join our discord server https://discord.gg/tSqtvHUJzE.

@Madhavdua have you joined this server? Because due to my inactivity, this project is moving in a loooop. So, join it to get notify with the project upcoming changes and guidelines.

**Comment by Madhavdua** [comment]
Hey @Avdhesh-Varshney i have integrated new feature and the quote section is not displayed yet, it may be added in future updates. 
For now, it is the final update, **it can be merged safely**, once previewed.
Screenshot 2024-06-19 222221
Screenshot 2024-06-19 222259

**Comment by Avdhesh-Varshney** [comment]
@Madhavdua leave it. Start working on other new issues and move with the flow of project.

**Pull request closed without merging**